src/tasks/processIssueComment/probot-processIssueComment.js

```diff
@@ -133,14 +162,18 @@ async function probotProcessIssueComment({ context, commentReply, analytics }) {
     analytics.track('processComment', {
         commentBody: commentBody,
     })
-    const { who, action, contributions } = parseComment(commentBody)
+    const { action, contributors } = parseComment(commentBody)
 
     if (action === 'add') {
-        analytics.track('addContributor', {
+        analytics.track('addContributors', {
             who: commentBody,
-            contributions,
+            contributors,
         })
-        const safeWho = getSafeRef(who)
+
+        const whos = contributors.map(({ who }) => who).join('--')
+        const safeWho = getSafeRef(whos)
+        // TODO: max length on branch name?
```

Comment:
If we were to have a max length, what would it be?

Reply:
I'd suggest 255 chars. While it's true that people could limit themselves to using GitHub's ui for everything, and with recent changes, it's possible to [renaming existing branches](https://github.com/github/renaming#renaming-existing-branches), branch names are generally stored in file systems and can be subject to the whims of file systems, some of whom don't particularly like very long file names (I'm not suggesting an 11 character limit...).

I suspect that would give you ~16-20 contributors, which is quite a lot.

Reply:
@jsoref That makes sense, yeah 255 chars seems good to me.